src/main/webapp/map.html

```diff
@@ -1,9 +1,11 @@
 <!DOCTYPE html>
 <html>
-<head>  
+<head>
     <title>Hello Google Maps</title>
+    <link rel = "stylesheet" href = "/css/main.css">
+    <link rel = "stylesheet" href = "/css/user-page.css">
```

Comment:
why do you use the user-page.css?

Reply:
Good question, I thought this would style it to look like the same colors as the rest of the page. I am not really sure how css works